src/main/java/com/spotify/docker/client/messages/NetworkSettings.java

```diff
@@ -98,32 +144,59 @@ static NetworkSettings create(
         .ports(portsCopy.build())
         .macAddress(macAddress)
         .networks(networks)
+        .endpointId(endpointId)
+        .sandboxId(sandboxId)
+        .sandboxKey(sandboxKey)
+        .hairpinMode(hairpinMode)
+        .linkLocalIpv6Address(linkLocalIpv6Address)
+        .linkLocalIpv6PrefixLen(linkLocalIpv6PrefixLen)
+        .globalIPv6Address(globalIPv6Address)
+        .globalIPv6PrefixLen(globalIPv6PrefixLen)
+        .ipv6Gateway(ipv6Gateway)
         .build();
   }
 
-  public static Builder builder() {
+  private static Builder builder() {
     return new AutoValue_NetworkSettings.Builder();
   }
 
   @AutoValue.Builder
-  public abstract static class Builder {
+  abstract static class Builder {
+
+    abstract Builder ipAddress(String ipAddress);
+
+    abstract Builder ipPrefixLen(Integer ipPrefixLen);
+
+    abstract Builder gateway(String gateway);
+
+    abstract Builder bridge(String bridge);
+
+    abstract Builder portMapping(Map<String, Map<String, String>> portMapping);
+
+    abstract Builder ports(Map<String, List<PortBinding>> ports);
+
+    abstract Builder macAddress(String macAddress);
+
+    abstract Builder networks(Map<String, AttachedNetwork> networks);
+
+    abstract Builder endpointId(final String endpointId);
 
-    public abstract Builder ipAddress(String ipAddress);
+    abstract Builder sandboxId(final String sandboxId);
 
-    public abstract Builder ipPrefixLen(Integer ipPrefixLen);
+    abstract Builder sandboxKey(final String sandboxKey);
 
-    public abstract Builder gateway(String gateway);
+    abstract Builder hairpinMode(final Boolean hairpinMode);
 
-    public abstract Builder bridge(String bridge);
+    abstract Builder linkLocalIpv6Address(final String linkLocalIpv6Address);
```

Comment:
Change to `IP` here and below if corresponding abstract method is changed to `IP` to be consistent.